design-documents/graph-ql/coverage/negotiableQuotes.graphqls

```diff
@@ -0,0 +1,361 @@
+type Query {
+
+    getNegotiableQuote(
+        id: ID!
+    ): NegotiableQuote @doc(description:"Get negotibale quote")
+
+    getAllNegotiableQuotes: [NegotiableQuote]! @doc(description: "Get all negotiable quotes")
```

Comment:
`getAllNegotiableQuotes: [NegotiableQuote]!`

This definition allows nulls to exist inside the list. Is there a case where we expect we'll be able to get _some_ but not _all_ results?